artix7/primitives/slicel/N5ff/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+make_file_target(FILE ntemplate.N5ff.model.xml)
+
+make_file_target(FILE ntemplate.N5ff.sim.v SCANNER_TYPE verilog)
+
+make_file_target(FILE ntemplate.N5ff.pb_type.xml)
+n_template(NAME N5ff.model.xml PREFIXES a;b;c;d SRCS ntemplate.N5ff.model.xml)
+
+n_template(
+  NAME N5ff.pb_type.xml
+  PREFIXES a;b;c;d
```

Comment:
Why does this use semicolons while `mux_gen` uses commas?

Reply:
Semicolons in CMake are special, they are the list divider.  So in this case PREFIXES is a list that CMake operates on (eg foreach(PREFIX ${PREFIXES}).  The commas are passed literally to mux_gen.  We could convert all lists to semicolons, and then convert the list to commas later, but that doesn't seem excessively valuable.

Reply:
I feel like the difference is going to bite me in the future :-P

Reply:
Agreed :( CMake's syntax is not pretty.  If this becomes and issue in the future, I can convert all lists in arguments to semi-colons and convert to commas when needed.

Reply:
Lets log an issue and plan to do this? It can be a good intro task?

Reply:
#219